Confus/Confus/Confus.vcxproj.filters

```diff
@@ -149,7 +153,11 @@
     <ClInclude Include="Weapon.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+<<<<<<< HEAD
+    <ClInclude Include="Health.h">
+=======
     <ClInclude Include="Flag.h">
+>>>>>>> refs/remotes/origin/develop
```

Comment:
Please fix this, these are remaints of a merge conflict. In fact, the merge conflict still exists